exporters/jaeger/Gemfile.lock

```diff
@@ -0,0 +1,72 @@
+PATH
```

Comment:
I notice we explicitly ignore the `api/Gemfile.lock` in the gitignore.

Should we make this more general and ignore this lock file as well?